dubbo-common/src/main/java/org/apache/dubbo/common/utils/StringUtils.java

```diff
@@ -718,4 +720,16 @@ public static String toArgumentString(Object[] args) {
         }
         return buf.toString();
     }
+
+    public static String simpleClassName(Class<?> clazz) {

```

Comment:
I think we should add a class Named ClassUtils or use org.apache.commons.lang3.ClassUtils.